x-pack/solutions/observability/plugins/streams/kibana.jsonc

```diff
@@ -17,7 +17,8 @@
       "usageCollection",
       "licensing",
       "taskManager",
-      "alerting"
+      "alerting",
+      "inference",
```

Comment:
not really related to this PR, but that looks like a bunch of dependencies that we can remove? taskManager, encryptedSavedObject, maybe even alerting although I guess we have some references to that in the AssetService